api-report/file-driver.api.md

```diff
@@ -69,7 +69,7 @@ export class FileDocumentServiceFactory implements IDocumentServiceFactory {
 // @public (undocumented)
 export const FileSnapshotWriterClassFactory: <TBase extends ReaderConstructor>(Base: TBase) => {
     new (...args: any[]): {
-        blobsWriter: Map<string, ArrayBufferLike>;
+        blobsWriter: Map<string, ArrayBuffer | SharedArrayBuffer>;
```

Comment:
Surprised this didn't get fixed earlier...

Reply:
Broke PR build, maybe it's unstable?  I'm undoing for this change, might be something deeper to look into here.